site/guide/model-workflows/manage-model-workflows.qmd

```diff
@@ -18,19 +18,41 @@ Review workflows associated with your models, and initiate or abort workflows co
 
 ## Review model workflows
 
-{{< include _review-model-workflows.qmd >}}
+### Review active workflows
+
+{{< include _review-active-workflows.qmd >}}
+
+### Review workflow history
+
+Review a timeline of workflows running or previously run on your model:
+
+1. In the left sidebar, click **{{< fa cubes >}} Inventory**.
+
+2. Select a model or find your model by applying a filter or searching for it.[^4]
+
+3. On the landing page of your model, locate the Workflows section. If workflows are running or have previously run on your model, you can:
+
+    - Filter the view using the **[show]{.smallcaps}** drop-down menu to display only runs from specific workflows.
+
+    - Click **{{< fa arrow-right-arrow-left >}} See All Workflows** to see records of workflows by status:
+        - **Active** — Workflows currently running.
+        - **Scheduled** — Workflows slated to run in the future.
+        - **Waiting** — Workflows in a wait state.[^5]
```

Comment:
yes, it gets there when workflow reaches a "Wait" step and its waiting for the specified date.